File: printplan-api/Models/Core/Printer.cs
using System.ComponentModel.DataAnnotations;

namespace printplan_api.Models.Core;

public class Printer
{
    [Key]
    public int Id { get; set; }
    public string Name { get; set; }
    public float PrinterSpeed { get; set; }
    public float PreheatingDuration { get; set; }

    public Printer(){}
}

File: printplan-app/Models/Printer.js
class Printer extends Realm.Object {
    static schema = {
        name: 'Printer',
        properties: {
            _id: 'objectId',
            name: 'string',
            preheatingDuration: 'Decimal128',
            printerSpeed: 'Decimal128'
        },
        primaryKey: '_id'
    }
}